db/schema.rb

```diff
@@ -167,7 +168,8 @@
   add_index "users", ["team_id"], name: "index_users_on_team_id", using: :btree
   add_index "users", ["team_role_id"], name: "index_users_on_team_role_id", using: :btree
 
-  add_foreign_key "bonus_credits", "teams"
+  add_foreign_key "bonus_reserves", "games"
+  add_foreign_key "bonus_reserves", "teams"
```

Comment:
I added in this file some missing index (bonus_reserves.game_id) and foreign keys (game_id.games and game_id.teams).